share/module_check_a_mundo.F

```diff
@@ -2271,7 +2328,8 @@ END FUNCTION bep_bem_ngr_u
       DO i = 1, model_config_rec % max_dom
          IF ( .NOT. model_config_rec % grid_allowed(i) ) CYCLE
          IF ( model_config_rec%mp_physics(i) .EQ. THOMPSONAERO ) THEN
-            IF ( model_config_rec%use_aero_icbc .AND. model_config_rec%scalar_pblmix(i) .NE. 1 ) THEN
+            IF ( (model_config_rec%use_aero_icbc .OR. model_config_rec%use_rap_aero_icbc) &
+                                      .AND. model_config_rec%scalar_pblmix(i) .NE. 1 ) THEN
```

Comment:
@twjuliano 
Tim,
It seemed like you were changing use_aero_icbc to something else, maybe aer_init_opt?

Reply:
@davegill Yes, I am using aer_init_opt as a replacement for use_aero_icbc and use_rap_aero_icbc. I did not use it here because my addition in check_a_mundo happens after this call. I can move my call before this one and swap out for aer_init_opt if you would prefer that.